mifosng-android/src/main/java/com/mifos/mifosxdroid/passcode/PassCodeActivity.java

```diff
@@ -84,23 +82,19 @@ public void savePassCode() {
         }
     }
 
-    @OnClick(R.id.btn_login)
-    public void loginUsingPassCode() {
-
-        if (!isInternetAvailable()) {
-            return;
-        }
-
-        if (counter == 3) {
-            Toaster.show(clRootview, R.string.incorrect_passcode_more_than_three);
-            PrefManager.clearPrefs();
```

Comment:
@mayank-kgp you did not clear the Passcode from Preference, It will cause to reopen Passcode activity If the user will not log in and close the app.